### ISSUE TITLE
searching for PMID: 25417150

### ISSUE THREAD

**Issue by deniseOme** [opentargets/library#19]
Is it WIP?

<img width="951" alt="screen shot 2017-06-29 at 19 11 22" src="https://user-images.githubusercontent.com/6472381/27703285-bff24e92-5cfe-11e7-8044-d8bd83069f21.png">

Or we believe this is not a use case?

**Comment by emepyc** [comment]
you can search for PMIDs, but just use the id (ie, no prefixing with `PMID:`)

**Comment by deniseOme** [comment]
How about DOIs?
e.g. 10.1038/nrneph.2016.107

https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1038%2Fnrneph.2016.107

EuropePMC does not support them, but PubMed does.

**Comment by emepyc** [comment]
Indeed, we have them and we plan to have a link directly to the DOI
